plugins/usertiming.js

```diff
@@ -0,0 +1,93 @@
+/**
+\file usertiming.js
+Plugin to collect metrics from the W3C User Timing API.
+For more information about User Timing,
+see: http://www.w3.org/TR/user-timing/
+*/
+
+(function() {
+
+	BOOMR = BOOMR || {};
+	BOOMR.plugins = BOOMR.plugins || {};
+
+	/**
+	 * @returns String compressed user timing data that occurred since the last call
+	 */
+	function getUserTiming() {
+		var now = BOOMR.now();
+
+		map = UserTimingCompression.getCompressedUserTiming(impl.options);
+		res = UserTimingCompression.compressForUri(map);
+		impl.options["from"] = now;
```

Comment:
Could there be a situation in which setting `options["to"]=now` would be different than not using `options["to"]` ? We're calling `now` and `getEntriesByType` synchronously, can something call `w.p.mark` or `w.p.measure` between those calls ?